### DISCUSSION TITLE
need help activating Student Education Pack benefits, have been approved for 3days

### DISCUSSION THREAD

**Discussion by Jamkris** [community/community#147354] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,
Hello! I saw in another post that if your student pack is not activated after 3 days, you should contact support. I am approaching the 3rd day and my application has already been accepted, so I am looking for support help. What should I do?

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/4600c80e-475e-422d-a102-84d9c1e75bcc" />

**Comment by davevad93** [comment, marked as answer]
Hi @Jamkris ,

Don't worry you don't have to do anything else, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Jamkris was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Jamkris** [reply]
Thank you for answer. I'll wait.
